schemas/rum/error-schema.json

```diff
@@ -47,6 +47,37 @@
               "description": "Stacktrace of the error",
               "readOnly": false
             },
+            "causes": {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏It could be nice to add causes example in the [sample](https://github.com/DataDog/rum-events-format/blob/master/samples/rum-events/error.json)